torch/_inductor/codegen/triton.py

```diff
@@ -1171,11 +1173,27 @@ def __init__(self, line, var, mask, size_map):
                 self.size_map = size_map
 
             def __call__(self):
-                # The conditions need to be in parens because of Python's operator precedence.
-                # It'd be less # error-prone to use and/or/not, which is suported by triton
-                size = self.size_map[(self.var, self.mask)]
-                cond = f"(0 <= {self.var}) & ({self.var} < {size})"
-                cond_print = f"0 <= {self.var} < {size}"
+                size, size_str = self.size_map[(self.var, self.mask)]
+
+                assert_min = (var.bounds.lower < 0) == sympy.true
+                assert_max = (var.bounds.upper >= size) == sympy.true
```

Comment:
This is capturing `var` from the parent scope, you want `self.var`.
```suggestion
                assert_min = (self.var.bounds.lower < 0) == sympy.true
                assert_max = (self.var.bounds.upper >= size) == sympy.true
```

Reply:
missed that one. Fixed now.